### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by jrd2017** [nickjj/docker-django-example#35] [General]
I'm new to Django on Docker. My question is: Do I need to use the command `django-admin startproject jrdapp ` on src file? or should I assume that the docker-compose creates the project? 

Thank you for such a complete repository.

**Comment by nickjj** [comment]
Hi,

The project is already created. If you want to rename it, you can follow this from the docs: https://github.com/nickjj/docker-django-example#running-a-script-to-automate-renaming-the-project

**Reply by jrd2017** [reply]
Thank you!! I kind of assumed it was created after reading one of the scripts.  GReat JOB with the repository.  :)
